stories/qa/tags/tags.stories.html

```diff
@@ -24,6 +24,25 @@ <h2>Tags</h2>
 	</div>
 </section>
 
+<!-- With icon -->
+<section class="contentSection">
+	<h2>With icon</h2>
+	<div class="u-displayFlex u-alignItemsFlexStart u-flexWrapWrap u-gapXS">
+		<span class="tag palette-product"
+			><span class="lucca-icon icon-heart" aria-hidden="true"></span> <span class="tag-content">Text</span></span
```

Comment:
Adding a `tag-content` seems a bit overkill in my opinion. It would also required extra conditions on Angular component.

Reply:
This is necessary for underlining when tags are clickable.